library/core/src/char/methods.rs

```diff
@@ -894,6 +894,14 @@ impl char {
     /// characters, and `No` for other numeric characters) are specified in the [Unicode Character
     /// Database][ucd] [`UnicodeData.txt`].
     ///
+    /// This method doesn't cover everything that could be considered a number, e.g. ideographic numbers.
+    /// If you want everything including characters with overlapping purposes then you might want
+    /// to use a unicode or language-processing library that exposes the appropriate character
+    /// properties instead of looking at the unicode categories.
+    ///
+    /// If you simply want to parse western arabic numerals (0-9) or base-N you'll
+    /// want `is_ascii_digit` or `is_digit` instead.
```

Comment:
```suggestion
    /// If you want to parse ASCII digits (0-9) or ASCII base-N, use
    /// `is_ascii_digit` or `is_digit` instead.
```

Let's avoid trying to define precisely what 0-9 are beyond "ASCII"; that seems like a potential minefield.

Reply:
Thanks! this make the docs more general and avoid corner case!